source/javascripts/components/GuidedOnboarding/types.ts

```diff
@@ -12,3 +13,11 @@ export interface SubStep {
     content: ReactNode;
     href?: string;
 }
+
+export enum BuildStatus {
```

Comment:
This is absolutely just a preference, but I'm curious what others think about it. I personally prefer objects with `as const` vs enums. 

<img width="898" alt="Screenshot 2022-10-24 at 10 19 05" src="https://user-images.githubusercontent.com/107245372/197483303-af9f18e8-7849-47b3-bd8e-41ae9913c8eb.png">

Both the build output and the typing make more sense to me.

Reply:
I like this pattern, updated